manifests/profile/master/postgres.pp

```diff
@@ -18,6 +18,7 @@
   Variant[String,Tuple[String, Integer]] $postgres_host = $facts['networking']['fqdn'],
   String[2] $query_interval                             = lookup('puppet_metrics_dashboard::pg_query_interval'),
   Integer[1] $port                                      = 5432,
+  Array[String] $databases                              = ['pe-puppetdb','pe-rbac','pe-activity','pe-classifier'],
```

Comment:
Please set this to be an array of non-empty strings by doing something like this:

```suggestion
  Array[String[1]] $databases                              = ['pe-puppetdb','pe-rbac','pe-activity','pe-classifier'],
```

Also, please add docs to the comments section above for this new parameter.

Reply:
Done!